mne/io/curry/tests/test_curry.py

```diff
@@ -64,27 +44,30 @@ def bdf_curry_ref():
 @pytest.mark.parametrize("preload", [True, False])
 def test_read_raw_curry(fname, tol, preload, bdf_curry_ref):
     """Test reading CURRY files."""
-    raw = read_raw_curry(fname, preload=preload)
+    with pytest.raises(RuntimeWarning):  # TODO change way to add montage in curry.py!
```

Comment:
With some suitable `match` regex string this should work.
```
    with pytest.warns(RuntimeWarning, match="..."):  # TODO change way to add montage in curry.py!
```
Warnings aren't `raise`d they are emitted via the warnings module, and for some reason this only fails on older Python. But it should fix the issue.

You also have some style issues according to the CIs

Reply:
> but `mne-tools.mne-python` still fails

yes, the "style" issue is what im asking about ;)

its not actually style, but that this CI can't import curryreader 

thanks for this other suggestion! though this one should disappear after my next planned refactoring step, anyway (the warning is thrown by smth I'll get rid of)